include/stumpless/priority.h

```diff
@@ -0,0 +1,68 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2022 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @file
+ * Priority value (PRIVAL) represents both the Facility and Severity values.
+ *
+ * @since release v2.1.0
```

Comment:
update this release to the current version, v2.2.0

Reply:
Ok, but I'm a bit confused. I can see in the github panel that current release is v2.1.0:
![image](https://github.com/goatshriek/stumpless/assets/144987715/12585e14-987b-41ca-97a1-8b66e97fee1f)

Reply:
Sorry, I should have said the current in work version. Since v2.1.0 is already out, we can't add new things to it, so new features go into the next scheduled release, which is v2.2.0 right now.